packages/core-utils/src/itinerary.js

```diff
@@ -466,10 +466,14 @@ export function getTransitFare(fareComponent) {
   let digits = 2;
   let transitFare = 0;
   let symbol = "$";
+  let currencyCode = "USD";
   if (fareComponent) {
-    digits = fareComponent.currency.defaultFractionDigits;
+    ({
+      currencyCode,
+      defaultFractionDigits: digits,
+      symbol
+    } = fareComponent.currency);
```

Comment:
Huh, I never knew this was possible! Might be useful to include a comment with link to the explanation for this syntax: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#assignment_without_declaration
```suggestion
    // Assign values without declaration. See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#assignment_without_declaration
    ({
      currencyCode,
      defaultFractionDigits: digits,
      symbol
    } = fareComponent.currency);
```